### PR TITLE
Added PUT Conversation metadata action

### PR DESCRIPTION
**Pull request by mjabali**

### CODE REVIEW

**Comment by dylburger** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

File: components/symbl_ai/symbl_ai.app.mjs [hunk 1]
```diff
@@ -225,5 +225,17 @@ export default {
...
+      conversationId,
+      data,
+    }) {
+      return this.makeRequest({
+        $,
+        method: "put",
+        path: `/conversations/${conversationId}`,
+        data,
+      });
+    },
```
Comment (jcortes) [inline components/symbl_ai/symbl_ai.app.mjs:239]:
```suggestion
    async putConversation({
      conversationId, ...args
    }) {
      return this.makeRequest({
        method: "put",
        path: `/conversations/${conversationId}`,
        ...args,
      });
    },
```

**Review by jcortes — CHANGES_REQUESTED** [review]
@mjabali Thanks for this contribution. It looks good. Just a couple of requested changes regarding refactoring and `prop` type and it's done!

File: components/symbl_ai/symbl_ai.app.mjs (hunk 1 above)
Comment (mjabali) [reply components/symbl_ai/symbl_ai.app.mjs]: Thanks. Addressed in f6b7f8b3.

File: components/symbl_ai/actions/put-conversation/put-conversation.mjs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  props: {
+    symblAIApp,
+    conversationId: {
+      propDefinition: [
+        symblAIApp,
+        "conversationId",
+      ],
+    },
+    metadata: {
+      type: "string",
```
[the remark this answers is not in the document]
Comment (mjabali) [reply components/symbl_ai/actions/put-conversation/put-conversation.mjs]: Thanks. Addressed in f6b7f8b3.

**Review by jcortes — APPROVED** [review]
LGTM! Ready for QA

**Pull request merged**